src/pkg/spinner/spinner.go

```diff
@@ -1,5 +1,11 @@
 package spinner
 
+import (
+	"context"
+	"fmt"
+	"time"
+)
+
 var SpinnerChars = `-\|/`
```

Comment:
`const` or perhaps we can find a sexier spinner and use that? https://stackoverflow.com/questions/2685435/cooler-ascii-spinners

Reply:
I have a stash on my local with this stuff. I'll put up a separate PR with ability to select spinners.